src/services/http/get-token-from-cookie.ts

```diff
@@ -0,0 +1,27 @@
+export function getTokenFromCookie(): string | null {
```

Comment:
To work with cookies, use a simple library `nookies`. If you need to track cookie changes, then use `react-cookie`